README.md

```diff
@@ -74,19 +85,18 @@ Some suggestions of things we might touch on include:
 
 ## Annual Leave
 
-- Expect to receive regular gentle nudges to ensure you are taking all the annual leave assigned each year - staff who barely take any leave will likely burn out as a result. I take these matters very seriously primarily in concern for the mental and physical wellbeing of my staff but also from a company point-of-view; exhausted, jaded staff are unlikely to deliver anything close to their potential quality and volume of work.
-- Please ensure (outside of unusual family circumstances) you are spreading your annual leave out over the holiday year - those who reach December with nearly a month's leave to take are both putting their wellbeing at risk and leave the company in the awkward position of having critical staff unavailable for extended periods of time.
+- Expect to receive regular gentle nudges to ensure you are taking all the annual leave assigned each year - staff who barely take any leave will likely burn out as a result. I take these matters very seriously primarily in concern for the mental and physical well-being of my staff but also from a company point-of-view; exhausted, jaded staff are unlikely to deliver anything close to their potential quality and volume of work.
```

Comment:
This is the only line which didn't read well to me - partly because of the long second sentence and partly because "in concern for" doesn't feel quite right. This might be changing the meaning slightly too much, but you could try something like:
- Expect to receive regular gentle nudges to ensure you are planning ahead to take all your annual leave. I take these matters very seriously, because taking leave is important for your mental and physical wellbeing. Avoiding burn-out is also an important part of maintaining a high performing team which benefits both you and the company.

Reply:
Nice, thanks!